onebusaway-android/src/main/java/org/onebusaway/android/util/RegionUtils.java

```diff
@@ -464,7 +464,6 @@ public static String formatOtpBaseUrl(String baseUrl) {
                         c.getString(12),             // OTP Base URL
                         c.getString(13),              // OTP Contact Email
                         c.getInt(14) > 0,           // Supports Otp Bikeshare
-                        c.getInt(15) > 0,            // Supports Embedded Social
```

Comment:
Per previous comment, let's leave the Regions API response intact in the app for now.